# HG changeset patch
# User Joel Marcey <joelm@fb.com>
# Date 1452027344 28800
#      Tue Jan 05 12:55:44 2016 -0800
# Node ID f77188bb2a4f0ef67edbcb6ffc2443b680592f30
# Parent  8c9846efd10d1148539d56658439c59edd68ddf7
[hhvm][oss] Update HHVM third-party submodule SHA after Squangle update

Summary:
This was pulled in: https://github.com/hhvm/hhvm-third-party/pull/99

Have HHVM's third-party submodule point to that.

Test Plan:
OSS still builds and tests pass

@override-unit-failures

Reviewers: fredemmott, sgolemon, lins

Reviewed By: lins

Subscribers: hphp-diffs@

Differential Revision: https://phabricator.fb.com/D2803830

Signature: t1:2803830:1452021526:79ec5123c7b501b4c4882cd52811dad132572e8e

diff --git a/fbcode/hphp/facebook/third-party-rev.txt b/fbcode/hphp/facebook/third-party-rev.txt
--- a/fbcode/hphp/facebook/third-party-rev.txt
+++ b/fbcode/hphp/facebook/third-party-rev.txt
@@ -1,1 +1,1 @@
-Subproject commit 6d9dffd0233c53bb83e4daf5475067073df9cdca
+Subproject commit ae031dcc9594163f5b0c35e7026563f1c8372595
